dapr/proto/runtime/v1/dapr.proto

```diff
@@ -672,4 +682,33 @@ message UnlockResponse {
   }
 
   Status status = 1;
-}
\ No newline at end of file
+}
+
+message WorkflowReference {
+  string InstanceID = 1;
+}
+
+message GetWorkflowRequest {
+  WorkflowReference WorkflowReference  = 1;
+  string WorkflowType = 2;
+  string WorkflowComponent = 3; 
+}
+
+message GetWorkflowResponse {
+  WorkflowReference WorkflowReference  = 1;
+  google.protobuf.Timestamp StartTime = 2;
```

Comment:
I don't believe we use `google.protobuf.Timestamp` anywhere, but it's a struct... Can we use int64 and pass a UNIX timestamp? Unless nanoseconds matter?